### ISSUE TITLE
support - dot like input

### ISSUE THREAD

**Issue by softmarshmallow** [adar2378/pin_code_fields#60]
not a circle put "·" dot like pincode input.
is there current way to implement this shape? since the shape cannot be fully customized.

**Comment by adar2378** [comment]
I see, for the obscure inputs?

**Comment by softmarshmallow** [comment]
Yes, in my use case the app called Zenly uses this kind of input on onboarding process, for phone sms verification

**Comment by adar2378** [comment]
For SMS verification? Are you sure my friend that they are obscure inputs? Obscure inputs are used for things like passwords. It could be just the inputs that have not filled yet the type of thing.

**Comment by softmarshmallow** [comment]
zenly designs
https://mobbin.design/apps/zenly

the screen reference
https://mobbin.design/apps/zenly#60O0DgyKSG4ps8LZIoll


it looks like obscure input, but only the place holder is. and if you tye into it, it just will display as-is

**Comment by adar2378** [comment]
I see, thanks, brother. Exactly what I thought. I'm a bit busy right now. If you can add this style then it would be very nice. Or I'll try this weekend.

**Comment by softmarshmallow** [comment]
alright, i'll take a look. how it can be done. thanks

**Comment by mwawrusch** [comment]
hi guys, any update on this - I would need it too. Thanks a ton!!!
